### DISCUSSION TITLE
New Branding Requests

### DISCUSSION THREAD

**Discussion by Allar** [jpanther/lynx#4] [Ideas]
Hi! Thanks for making this theme, I'm using it for an internal home and landing page of sorts.

I'm really bad at making logo/color choices, but I'd love if some additional ones were made for:

ClickUp
Gmail
Notion
Bitwarden
Cloudflare
AWS
Justworks

I've made some attempts at some svgs but they don't look as great as the ones provided and I'm not sure how to fix them .svg wise.

[logos.zip](https://github.com/jpanther/lynx/files/7480926/logos.zip)

If anyone seeing this is willing to do a pass, I humbly request the ones above to be added.

**Comment by jpanther** [comment]
Hey @Allar, thanks for the request. The logos in the theme come from FontAwesome so I'd prefer to only bundle ones from there. Some of the brands you've mentioned are already on there... https://fontawesome.com/icons?d=gallery&p=1&s=brands

However, the theme supports providing your own icons, and in your case this is probably what you want to do. You can simply place them in `/assets/icons/` and then reference them in your config. You can also create a custom stylesheet at `/assets/css/custom.css` and set colours for your new links.

For instance, to use Cloudflare as an example...

1. Place the SVG [icon](https://fontawesome.com/icons/cloudflare?style=brands) at `/assets/icons/cloudflare.svg`
2. Create `/assets/css/custom.css` and add a rule as follows:  

    ```css
    .link-cloudflare { 
        background-color: #F38121;
    }
    ```

3. Finally, in your `config.toml` you can make a link: 
    
    ```toml
    cloudflare = { href = "https://cloudflare.com", text = "Cloudflare" }
    ```

When you build your site, the resulting button looks like this:
![Screen Shot 2021-11-05 at 17 49 51](https://user-images.githubusercontent.com/4462786/140470075-bf5da937-48eb-4dc4-97c5-a6b280e10c84.JPG)
